index.js

```diff
@@ -1,5 +1,9 @@
 /*! load-script2. MIT License. Feross Aboukhadijeh <https://feross.org/opensource> */
 module.exports = function loadScript2 (src, attrs, parentNode) {
+  const node = parentNode || document.head || document.getElementsByTagName('head')[0]
+  const existingScript = node.querySelector(`:scope > script[src="${src}"]`)
+  if (existingScript) return existingScript
```

Comment:
@feross this allows to cache the added script

it can be useful when you render more than once a component that uses loadScript